reference/fleet/providers.md

```diff
@@ -53,50 +76,40 @@ Context providers give the current context of the running {{agent}}, for example
 
 They can only provide a single key-value mapping. Think of them as singletons; an update of a key-value mapping results in a re-evaluation of the entire configuration. These providers are normally very static, but not required. A value can change which results in re-evaluation.
 
-Context providers use the Elastic Common Schema (ECS) naming to ensure consistency and understanding throughout documentation and projects.
+Context providers use the {{product.ecs}} naming to ensure consistency and understanding throughout documentation and projects.
 
 {{agent}} supports the following context providers:
 
-* [Local](/reference/fleet/local-provider.md)
-* [Agent Provider](/reference/fleet/agent-provider.md)
-* [Host Provider](/reference/fleet/host-provider.md)
-* [Env Provider](/reference/fleet/env-provider.md)
-* [Kubernetes Secrets Provider](/reference/fleet/kubernetes_secrets-provider.md)
-* [Kubernetes Leader Election Provider](/reference/fleet/kubernetes_leaderelection-provider.md)
+* [Local provider](/reference/fleet/local-provider.md)
+* [Agent provider](/reference/fleet/agent-provider.md)
+* [Host provider](/reference/fleet/host-provider.md)
+* [Env provider](/reference/fleet/env-provider.md)
+* [Kubernetes Secrets provider](/reference/fleet/kubernetes_secrets-provider.md)
+* [Kubernetes LeaderElection provider](/reference/fleet/kubernetes_leaderelection-provider.md)
```

Comment:
Are these obvious? What about short one-line descriptions for each link so readers know what they’re clicking into?

> Host provider – Supplies host-level metadata such as hostname and IP.